dark/fasta.py

```diff
@@ -93,12 +97,11 @@ def iter(self):
         Iterate over the sequences in self.file_, yielding each as an
         instance of the desired read class.
         """
-        first = True
-        for seq in SeqIO.parse(self.file_, 'fasta'):
-            read = self.readClass(seq.description, str(seq.seq))
-            if first:
+        checkAlphabet = self._checkAlphabet
```

Comment:
I do it because I don't want Python to have to look up `self._checkAlphabet` twice for each sequence in the `for` loop.  It shouldn't really be needed (and running under PyPy would make the inefficiency go away, I am pretty sure), but it's a habit I have (as do some others) to not make Python be quite so slow.